frontend/auto_linux.sh

```diff
@@ -0,0 +1,19 @@
+if [ ! -d dist ]; then
```

Comment:
No extra scripts should be required to install extra packages. Please use the npm package manager.